common_benchmark_suite/openxla/benchmark/comparative_suite/tf/scripts/generate_model_artifacts.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Runs `generate_model_artifacts.py` on all registered JAX models and saves
+# artifacts into the directory `/tmp/jax_models_<jax-version>_<timestamp>`.
+#
+# Once complete. please upload the output directory to
+# `gs://iree-model-artifacts/jax`, preserving directory name.
```

Comment:
nit: `jax` needs to be replaced with `tf`

Reply:
Fixed